tests/test_isosurface.py

```
@@ -0,0 +1,546 @@
+        resp_coarse.content, object_hook=msgpack_numpy.decode, raw=False
+    )
+
+    assert len(data_coarse["vertices"]) < len(data_fine["vertices"])
+
+
+@pytest.mark.asyncio
+async def test_isosurface_room_not_found(
+    iso_client: AsyncClient, iso_session: AsyncSession
+) -> None:
+    """GET isosurface for non-existent room returns 404."""
+    user, token = await create_test_user_in_db(iso_session)
```

Ruff (0.15.5)

[warning] Unpacked variable `user` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)